pages/interface/components/PasswordInput/index.js

```diff
@@ -44,12 +44,12 @@ export default function PasswordInput({ inputRef, id, name, label, errorObject,
     <FormControl id={id}>
       <FormControl.Label>{label}</FormControl.Label>
       <TextInput
-        trailingAction={
+        trailingVisual={
           <TextInput.Action
             aria-label={isPasswordVisible ? 'Ocultar a senha' : 'Visualizar a senha'}
             onClick={handlePasswordVisible}
             icon={isPasswordVisible ? EyeClosedIcon : EyeIcon}
-            sx={{ color: 'fg.subtle' }}
+            sx={{ color: 'fg.subtle', padding: '0' }}
```

Comment:
```suggestion
            sx={{ color: 'fg.subtle' }}
```

Não podemos deixar o padding zerado por questão de acessibilidade, pois a área clicável fica muito pequena.

Reply:
Entendi, realmente não pensei nesse detalhe. Show! 🤝